server/lib/payments.js

```diff
@@ -951,7 +951,7 @@ export const getHostFee = async (order, host = null) => {
 
   const hostFeePercent = await getHostFeePercent(order, host);
 
-  return calcFee(order.totalAmount - platformTip, hostFeePercent);
```

Comment:
Importing this fix from https://github.com/opencollective/opencollective-api/pull/8665